### ISSUE TITLE
Same reward when running the example notebook with TD3 algorithm

### ISSUE THREAD

**Issue by JojoLiu6697** [AI4Finance-Foundation/FinRL#935]
Hi,

When I run the example file 'Stock_NeurlPS2018_2_Train.ipynb', the actor loss and the function loss keeps decreasing while the reward keeps the same. I did not change any code in the example file. And I am sure I did not change the model or environment either since I run the code as soon as I installed the Fin-RL. Can you help me with that?

Thanks

**Comment by JojoLiu6697** [comment]
This time I changed the learning rate to 0.0005, but still, the reward keeps the same:
---------------------------------
| time/              |          |
|    episodes        | 4        |
|    fps             | 82       |
|    time_elapsed    | 140      |
|    total_timesteps | 11572    |
| train/             |          |
|    actor_loss      | 35       |
|    critic_loss     | 286      |
|    learning_rate   | 0.0005   |
|    n_updates       | 8679     |
|    reward          | 5.367151 |
---------------------------------
---------------------------------
| time/              |          |
|    episodes        | 8        |
|    fps             | 78       |
|    time_elapsed    | 293      |
|    total_timesteps | 23144    |
| train/             |          |
|    actor_loss      | 29.1     |
|    critic_loss     | 25.8     |
|    learning_rate   | 0.0005   |
|    n_updates       | 20251    |
|    reward          | 5.367151 |
---------------------------------
day: 2892, episode: 10
begin_total_asset: 1000000.00
end_total_asset: 3756541.57
total_reward: 2756541.57
total_cost: 1161.44
total_trades: 49164
Sharpe: 0.711
=================================
---------------------------------
| time/              |          |
|    episodes        | 12       |
|    fps             | 74       |
|    time_elapsed    | 462      |
|    total_timesteps | 34716    |
| train/             |          |
|    actor_loss      | 25.6     |
|    critic_loss     | 4.59     |
|    learning_rate   | 0.0005   |
|    n_updates       | 31823    |
|    reward          | 5.367151 |
---------------------------------
---------------------------------
| time/              |          |
|    episodes        | 16       |
|    fps             | 74       |
|    time_elapsed    | 618      |
|    total_timesteps | 46288    |
| train/             |          |
|    actor_loss      | 23.2     |
|    critic_loss     | 2.68     |
|    learning_rate   | 0.0005   |
|    n_updates       | 43395    |
|    reward          | 5.367151 |
---------------------------------

**Comment by JojoLiu6697** [comment]
Probably it has something to do with the environment? Check this:
[https://github.com/DLR-RM/stable-baselines3/issues/1026](url)

**Comment by JojoLiu6697** [comment]
It is the hyperparameter that matters. When I use different target_policy_noise and gamma, the reward changed
